creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -79,12 +91,27 @@ const healthCheckVerboseController = async (req) => {
   })
 }
 
+/**
+ * Controller for `health_check/fileupload` route
+ * Calls `healthCheckFileUploadService`.
+ */
+const healthCheckFileUploadController = async (req) => {
+  const err = req.fileFilterError || req.fileSizeError
+  if (err) {
+    throw new Error(err)
+  } else {
+    removeTrackFolder(req, req.fileDir)
+    return successResponse({ success: true })
```

Comment:
is it sufficient to only call `removeTrackFolder` in this else block? looking at the `fileManager` code, seems there are a few diff places errors are checked/thrown and we may want to call `removeTrackFolder` in a `try-catch` no matter what instead of only in the else case here. what do you think?

Reply:
Yes, I agree, there is a possibility of some garbage being partially stored. I've added a `try-catch` for this now.